ansible/roles/discourse/tasks/main.yml

```diff
@@ -14,5 +14,25 @@
   template:
     src: standalone.yml.j2
     dest: "{{ dis_path }}/containers/app.yml"
-  notify:
-    launcher rebuild
+  register: dis_template
+
+- name: Get Discourse container info
+  docker_container_info:
+    name: app
+  register: dis_ct
+
+- name: Print the status of the Discourse container
+  debug:
+    msg: "The container status is {{ dis_ct.container.State.Status }}"
+  when: dis_ct.exists
+
+- name: (Re)build Discourse container configuration
+  command: "{{ dis_path }}/launcher rebuild app"
+  when: not dis_ct.exists or dis_template.changed
+
+- name: Start Discourse container if it's not running
```

Comment:
I would make the comment more explicit like `Start Discourse container if it already exists but is not running`